modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,292 @@
+//. Restarts the node with the modified override file. Remove when content is SME approved.
+. Verify that the `br-ex` MAC address matches the MAC address for the interface that `br-ex` uses for the network connection.
+. Executes the `configure-ovs.sh` shell script that references the new interface definition.
+. Adds `br-ex` and `br-ex1` to the host node.
+. Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Optional: Create an interface connection that your secondary bridge, `br-ex1`, can use by completing the following steps. The example steps show the creation of a new bond and its dependent interfaces that are all defined in a machine configuration manifest file. The additional bridge uses the `MachineConfig` object to form a secondary bond interface.
++
+[IMPORTANT]
+====
+Do not use the Kubernetes NMState Operator to define or a `NodeNetworkConfigurationPolicy` (NNCP) manifest file to define the secondary interface.
+
+Also ensure that the secondary interface or sub-interfaces when defining a `bond` interface are not used by existing an `br-ex` OVN Kubernetes network deployment.
+====
++
+.. Create the following interface definition files. These files get added to a machine configuration manifest file so that host nodes can access the definition files. 
++
+.Example of the first interface definition file that is named `eno1.config`
+[source,ini]
+----
+[connection]
+id=eno1
+type=ethernet
+interface-name=eno1
+master=bond1
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second interface definition file that is named `eno2.config`
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond1
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second bond interface definition file that is named `bond1.config`
+[source,ini]
+----
+[connection]
+id=bond1
+type=bond
+interface-name=bond1
+autoconnect=true
+connection.autoconnect-slaves=1
+autoconnect-priority=20
+
+[bond]
+mode=802.3ad
+miimon=100
+xmit_hash_policy="layer3+4"
+
+[ipv4]
+method=auto
+----
++
+.. Convert the definition files to Base64 encoded strings by running the following command:
++
+[source,terminal]
+----
+$ base64 <directory_path>/en01.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/eno2.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/bond1.config
+----
+
+. Prepare the environment variables. Replace `<machine_role>` with the node role, such as `worker`, and replace `<interface_name>` with the name of your additional `br-ex` bridge name.
++
+[source,terminal]
+----
+$ export ROLE=<machine_role>
+----
++
+[source,terminal]
```

Comment:
We can remove the export for INTERFACE here - not required for this portion of the setup

Reply:
Thanks. Removed:

> +
[source,terminal]
----
$ export INTERFACE=<interface_name>
----